src/server/game/Entities/Player/Player.h

```diff
@@ -2534,8 +2561,8 @@ class Player : public Unit, public GridObject<Player>
         bool SetFeatherFall(bool apply, bool packetOnly = false);
         bool SetHover(bool enable, bool packetOnly = false);
 
-        bool CanFly() const { return m_movementInfo.HasMovementFlag(MOVEMENTFLAG_CAN_FLY); }
-
+        bool CanFly() const override { return m_movementInfo.HasMovementFlag(MOVEMENTFLAG_CAN_FLY); }
```

Comment:
You add `override` to this function. Now you need to do so with all the virtual functions in this class